src/pacemaker.c

```diff
@@ -120,7 +120,7 @@ static int pcmk_set_attr(struct ticket_config *tk, const char *attr, const char
 	int rv;
 
 	rv = snprintf(cmd, COMMAND_MAX,
-		 "crm_ticket -t '%s' -S '%s' -v '%s'",
+		 "crm_ticket -t '%s' -S '%s' --attr-value='%s'",
```

Comment:
Not entirely sure what you mean ;) Only `-v` is converted and I think it make sense to convert all `-v` to `--attr-value`, not only to keep code somehow consistent but mainly because this function is used for geostore and user can store string beginning with `-`. No matter what, if you think `-v ` is better here, I will change it.

Reply:
Got it :) I've tweaked commit message so worry part is gone.